File: Go/gob.go
package main
import (
        "fmt"
        "time"
)

func main() {
        var x string
        fmt.Println("Gob's program: y/n?")
        fmt.Scanf("%s", &x)
        if x == "y" {
                for {
                        fmt.Print("Penus ")
                        time.Sleep(100 * time.Millisecond)
                }
        }
}

File: PHP/gob.php
<?
	echo "Gob's Program: Y/N?\n";
	$handle = fopen("php://stdin", "r");
	$response = fgets($handle);
	fclose($handle);

	if (strtoupper(trim($response)) != "Y") {
		exit;
	}

	while (1) {
		echo "Penus ";
	}
?>